src/site/content/en/blog/build-for-webusb/index.md

```diff
@@ -0,0 +1,1014 @@
+devices and will allow the browser to access them as long as there is at least
+one unclaimed interface.
+
+### Windows
+
+The Windows driver model introduces an additional requirement. Unlike the
+platforms above the ability to open a USB device from a user application is not
+the default, even if there is no driver loaded. Instead there is a special
+driver, WinUSB, that needs to be loaded in order to provide the interface
+applications use to access the device. This can be done with either a custom
+driver information file (INF) installed on the system or by modifying the device
+firmware to provide the Microsoft OS Compatibility Descriptors during
+enumeration.
+
+#### Driver Information File (INF)
+
+A driver information file tells Windows what to do when encountering a device
+for the first time. Since the user's system already includes the WinUSB driver
+all that's necessary is for the INF file to associate your vendor and product ID
+with this new installation rule. The file below is a basic example. Save it to a
+file with the `.inf` extension, change the sections marked with "X", then right
+click on it and choose "Install" from the context menu.
+
+```ini
+[Version]
+Signature   = "$Windows NT$"
+Class       = USBDevice
+ClassGUID   = {88BAE032-5A81-49f0-BC3D-A4FF138216D6}
+Provider    = %ManufacturerName%
+CatalogFile = WinUSBInstallation.cat
+DriverVer   = 09/04/2012,13.54.20.543
+
+; ========== Manufacturer/Models sections ===========
+
+[Manufacturer]
+%ManufacturerName% = Standard,NTx86,NTia64,NTamd64
+
+[Standard.NTx86]
+%USB\MyCustomDevice.DeviceDesc% = USB_Install,USB\VID_XXXX&PID_XXXX
+
+[Standard.NTia64]
+%USB\MyCustomDevice.DeviceDesc% = USB_Install,USB\VID_XXXX&PID_XXXX
+
+[Standard.NTamd64]
+%USB\MyCustomDevice.DeviceDesc% = USB_Install,USB\VID_XXXX&PID_XXXX
+
+; ========== Class definition ===========
+
+[ClassInstall32]
+AddReg = ClassInstall_AddReg
+
+[ClassInstall_AddReg]
+HKR,,,,%ClassName%
+HKR,,NoInstallClass,,1
+HKR,,IconPath,%REG_MULTI_SZ%,"%systemroot%\system32\setupapi.dll,-20"
+HKR,,LowerLogoVersion,,5.2
+
+; =================== Installation ===================
+
+[USB_Install]
+Include = winusb.inf
+Needs   = WINUSB.NT
+
+[USB_Install.Services]
+Include = winusb.inf
+Needs   = WINUSB.NT.Services
+
+[USB_Install.HW]
+AddReg = Dev_AddReg
+
+[Dev_AddReg]
+HKR,,DeviceInterfaceGUIDs,0x10000,"{XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX}"
+
+; =================== Strings ===================
+
+[Strings]
+ManufacturerName              = "Your Company Name Here"
+ClassName                     = "Your Company Devices"
+USB\MyCustomDevice.DeviceDesc = "Your Device Name Here"
+```
+
+The `[Dev_AddReg]` section configures the set of DeviceInterfaceGUIDs for the
+device. Every device interface must have a GUID in order for an application to
+find and connect to it through the Windows API. Use the `New-Guid` PowerShell
+cmdlet or an online tool to generate a random GUID.
+
```

Comment:
"For development purposes the [Zadig tool](https://zadig.akeo.ie/) provides an easy interface for replacing the driver loaded for a USB interface with the WinUSB driver."

Reply:
Done in https://github.com/GoogleChrome/web.dev/pull/4240/commits/72c864368e12ee579d59eb241442c8a604f8d4a3